src/Symfony/Component/HttpKernel/EventListener/ErrorListener.php

```diff
@@ -12,10 +12,12 @@
 namespace Symfony\Component\HttpKernel\EventListener;
 
 use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel as PsrLogLevel;
```

Comment:
We don't need to alias this class anymore, do we?

Reply:
Ah, I've missed to remove that.